tests/perf/auto_perf_sheriffing/test_report_backfill_outcome.py

```diff
@@ -0,0 +1,121 @@
+from datetime import datetime, timedelta
+
+import pytest
+from tests import settings as test_settings
+from django.core.management import call_command
+
+from treeherder.perf.auto_perf_sheriffing.sherlock import Sherlock
+from treeherder.perf.models import BackfillNotificationRecord
+from treeherder.perf.exceptions import MaxRuntimeExceeded
+
+EPOCH = datetime.utcfromtimestamp(0)
+
+
+# TODO: Update tests so that the mock 'tc_notify_mock' works as expected
```

Comment:
Is there a bug for this? There are so many 'TODO' comments in the code base that aren't really dealt with later on (not your teams fault of course) :)

Reply:
I've added a new bug for this, thanks for your review.